spec/support/mock_proofer_helper.rb

```diff
@@ -0,0 +1,21 @@
+def expect_mock_proofer_matches_real_proofer(mock_proofer_class:, real_proofer_class:)
```

Comment:
Personally I might expect a method that starts with `expect_` to only include assertions like `expect(...)`, not itself include groupings and test cases. Should this be something we implement as `shared_examples` , or at least name like such e.g. `it_behaves_like_mock_proofer` ?

Reply:
I find Rspec's string key shared examples hard to track which is why I went with a plain method approach. How would you feel about `assert_mock_proofer_matches_real_proofer`?

But I'm also open to the `it_behaves_like_` name

Reply:
went with your suggestion in 4fac23543ef011c1b070007cfb440f298f699da3